com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,144 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
```

Comment:
Is all needed or can it be dri? all gives you controllers, webcams etc. but I don't think a code editor needs them?

Reply:
> Is all needed or can it be dri? all gives you controllers, webcams etc. but I don't think a code editor needs them?

Needs for usb, mostly for auth tokens like yubikey, and fido2 tokens